### PR TITLE
Switch to nc13-RC1 to nc14-dev

### PR DESCRIPTION
**Pull request by ChristophWurst**
FYI this means 0.7.9 is the last nc12-compatible release.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -114,9 +114,9 @@ script:
 matrix:
   include:
     - php: 5.6
-      env: "DB=mysql CORE_BRANCH=stable12 TEST_JS=TRUE"
+      env: "DB=mysql CORE_BRANCH=master TEST_JS=TRUE"
```
Comment (jancborchardt) [inline .travis.yml]: Should it not be stable13 now that we branched off?

Comment (ChristophWurst) [reply .travis.yml]: See line 25

File: .travis.yml [hunk 2]
```diff
@@ -113,10 +113,10 @@ script:
 
 matrix:
   include:
-    - php: 5.6
-      env: "DB=mysql CORE_BRANCH=stable12 TEST_JS=TRUE"
     - php: 7.0
-      env: "DB=mysql CORE_BRANCH=stable12 PACKAGE=TRUE"
+      env: "DB=mysql CORE_BRANCH=master TEST_JS=TRUE"
+    - php: 7.1
+      env: "DB=mysql CORE_BRANCH=master PACKAGE=TRUE"
```
Comment (rullzer) [inline .travis.yml]: Please also add tests against `CORE_BRANCH=stable13` if you still support that :wink: 

Also please add php7.2 tests

Comment (ChristophWurst) [reply .travis.yml]: Please see the full file. Stable13 had already been the default core branch and we test php56 to php72 :wink:

Comment (rullzer) [reply .travis.yml]: Ah my bad. Man I hate this weird travis syntax :P

**Pull request merged**